src/Entities/ExpOrb.h

```diff
@@ -8,14 +8,15 @@
 
 
 // tolua_begin
-class cExpOrb :
+class cExpOrb:
 	public cEntity
 {
-	typedef cEntity super;
-
-public:
 	// tolua_end
 
+	using Super = cEntity;
+
+public:  // tolua_export
```

Comment:
Goods and services to the European Union.

Reply:
tolua++ completely ignores anything not in the exported segments (as if it were `ifdef`'d out). So, without this, it would completely ignore the public specifier.